scripts/ipet-parse

```diff
@@ -92,6 +92,59 @@ argparser.add_argument("-v", "--validatedual", action = "store_true", default =
 argparser.add_argument("-g", "--gaptol", type = float, default = Experiment.DEFAULT_GAPTOL, help = "relative tolerance for primal and dual objective validation")
 argparser.add_argument("--docmode", action = "store_true", default = False, help = "print this help as restructured text")
 argparser.add_argument("--csv", action = "store_true", default = False, help = "print the data as csv")
+argparser.add_argument("-j", "--jobs", default = -1, type=int, help = "number of threads to use (-1 for all available threads)")
+
+def setup_experiment(arguments, verbose=False):
+    experiment = Experiment(validatedual = arguments.validatedual, gaptol = arguments.gaptol)
+    for reader in loader.loadAdditionalReaders(arguments.readers):
+
+        experiment.readermanager.registerReader(reader)
+        if verbose:
+            logger.info("Registered reader with name %s" % reader.getName())
+
+    for solver in loader.loadAdditionalSolvers():
+        experiment.readermanager.addSolver(solver)
+        if verbose:
+            logger.info("Registered solver with name %s" % solver.getName())
+
+    for solufile in loader.loadAdditionalSolufiles(arguments.solufiles):
+        experiment.addSoluFile(solufile)
+        if verbose:
+            logger.info("Imported solufile with name %s" % solufile)
+
+    return experiment
+
+def process_one_outfile(task : tuple):
+    # initialize an experiment
+    outfile, arguments = task
+    experiment = setup_experiment(arguments)
+
+    logger.info("Start parsing process of outfile '{}'".format(outfile))
+
+    experiment.addOutputFile(outfile)
```

Comment:
in the call of `process_one_outfile` around line 190 only the `.out` file is provided, what about the other types of files like `.err`, `.set` and `.meta`?

Reply:
You don't need to pass them, anyway, because the experiment is going to find files of the same base name internally such as a meta file, anyway. I would prefer to keep the logic that groups files together inside an experiment, which is what Rubberband uses, anyway. Can you think of any use case where this is needed here? Do you or anybody who's left in the group ever use ipet-parse from the command line?

Reply:
Oh, i was not aware that the experiment does this. Can you tell me where it loads different files with the same basename? 
Then the discussion here is resolved. 
I tell people that they should use ipet-parse on the command line. if they actually do this though i don't know ;)

Reply:
I realized that what I mentioned happens inside TestRun.py. A newly created TestRun object with an OUT file will automatically pick up a neighboring META file of the same basename, but not ERR or SET files.

I have modified ipet-parse now to group related OUT, SET, and ERR files together (by their path/basename, without file extension).